pkg/volume/portworx/portworx_util.go

```diff
@@ -70,12 +73,14 @@ func (util *PortworxVolumeUtil) CreateVolume(p *portworxVolumeProvisioner) (stri
 	if err != nil {
 		glog.V(2).Infof("Error creating Portworx Volume : %v", err)
 	}
+
+	glog.Infof("Successfully created Portworx volume for PVC: %v", p.options.PVC.Name)
 	return volumeID, requestGB, nil, err
 }
 
 // DeleteVolume deletes a Portworx volume
 func (util *PortworxVolumeUtil) DeleteVolume(d *portworxVolumeDeleter) error {
-	driver, err := util.getPortworxDriver(d.plugin.host)
+	driver, err := util.getPortworxDriver(d.plugin.host, false)
```

Comment:
Everywhere we add a naked `true` or `false` argument - we should also add a comment of the form `true /*localOnly*/`